alacritty_terminal/Cargo.toml

```diff
@@ -21,7 +21,7 @@ libc = "0.2"
 log = "0.4"
 parking_lot = "0.12.0"
 polling = "3.0.0"
-regex-automata = "0.3.6"
+regex-automata = "0.4.3"
```

Comment:
I've read the history and haven't found an obvious reason, but I'd suggest you to double check on that matter.